services/wallet/transfer/block_ranges_sequential_dao.go

```diff
@@ -0,0 +1,146 @@
+package transfer
+
+import (
+	"database/sql"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/status-im/status-go/services/wallet/bigint"
+)
+
+const (
+	firstBlockColumn = "blk_first"
+	lastBlockColumn  = "blk_last"
+	startBlockColumn = "blk_start"
+)
+
+type BlockRangeSequentialDAO struct {
+	db *sql.DB
+}
+
+type BlockRange struct {
+	Start      *big.Int // Block of first transfer
+	FirstKnown *big.Int // Oldest scanned block
+	LastKnown  *big.Int // Last scanned block
+}
+
+func NewBlockRange() *BlockRange {
+	return &BlockRange{Start: &big.Int{}, FirstKnown: &big.Int{}, LastKnown: &big.Int{}}
+}
+
+func (b *BlockRangeSequentialDAO) getBlockRange(chainID uint64, address common.Address) (blockRange *BlockRange, err error) {
+	query := `SELECT blk_start, blk_first, blk_last FROM blocks_ranges_sequential
+	WHERE address = ?
+	AND network_id = ?`
+
+	rows, err := b.db.Query(query, address, chainID)
+	if err != nil {
+		return
+	}
+	defer rows.Close()
+
+	if rows.Next() {
+		blockRange = NewBlockRange()
+		err = rows.Scan((*bigint.SQLBigInt)(blockRange.Start), (*bigint.SQLBigInt)(blockRange.FirstKnown), (*bigint.SQLBigInt)(blockRange.LastKnown))
+		if err != nil {
+			return nil, err
+		}
+
+		return blockRange, nil
+	}
+
+	return nil, nil
+}
+
+// TODO call it when account is removed
```

Comment:
Do we have any follow-up task to link with this TODO and ensure we don't leave dead data in the table?